### PR TITLE
Add Combo article

### PR DESCRIPTION
**Pull request by Pennek**
Resolves part of #3151

This article focuses on combo as groupings of hit objects. Another article on combos as a score multiplier is still needed

### CODE REVIEW

**Comment by cl8n** [comment]
i think this would make more sense as a "Combos" article rather than only combo colors? the color is a property of a combo, so it's difficult to explain in isolation (this article already has plenty of info about combos in general)

**Comment by Pennek** [comment]
Got it, will try rewriting it into this after i apply some reviews

**Comment by Pennek** [comment]
There's a lot of info on 'combos' in general on wiki already, but the current use of the word is very ambiguous as it refers to both combo as a score multiplier and combo as groupings of hit objects. Cleaning this up and separating the two will be for the better imo.

In this article I've focused on combo as groupings of hit objects, but another article on score multiplier is needed.

File: wiki/Beatmapping/Combo/en.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+tags:
+    - Combo Color
+    - Colorhaxing
+    - Colourhaxing
+    - Color Haxing
+    - Colour Haxing
+    - NC
```
Comment (MegaApplePi) [inline wiki/Beatmapping/Combo/en.md]:
```suggestion
  - Combo Color
  - Colorhaxing
  - Colourhaxing
  - Color Haxing
  - Colour Haxing
  - NC
```

File: wiki/Beatmapping/Combo/en.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+# Combo
+
+*For the element refered to as 'combo' in gameplay, see: [Score Multiplier](/wiki/Score)*
```
Comment (MegaApplePi) [inline wiki/Beatmapping/Combo/en.md]:
```suggestion
*Not to be confused with [Score Multiplier](/wiki/Score).*
```

File: wiki/Beatmapping/Combo/en.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Combos refer to groupings of [hit objects](/wiki/Hit_Objects) done according to musical representation. Groupings are most commonly done according to verses, measures, vocal phrases and/or instrument scripts. Different combos are separated by **New Combos**, or ***NCs***. New combos are made with the `New Combo` button (or the shortcut `Q`) on the right of the `Design` tab in the [beatmap editor](/wiki/Beatmap_Editor). How combos are displayed in gameplay depends on the currently enabled [gamemode](/wiki/Game_Modes).
+
+## Combo Colour
```
Comment (MegaApplePi) [inline wiki/Beatmapping/Combo/en.md]:
```suggestion
## Combo colour
```

File: wiki/Beatmapping/Combo/en.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+*For regulations surrounding combo colours, see [Ranking Criteria](/wiki/Ranking_Criteria)*
```
Comment (MegaApplePi) [inline wiki/Beatmapping/Combo/en.md]:
```suggestion
*For regulations surrounding combo colours, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
```

**Pull request merged**